src/Pim/Component/Catalog/Normalizer/Indexing/Product/PriceCollectionNormalizer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Pim\Component\Catalog\Normalizer\Indexing\Product;
+
+use Pim\Component\Catalog\AttributeTypes;
+use Pim\Component\Catalog\Model\ProductValueInterface;
+use Pim\Component\Catalog\ProductValue\PriceCollectionProductValue;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalizer for a price collection attribute
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class PriceCollectionNormalizer extends AbstractProductValueNormalizer implements NormalizerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, $format = null)
+    {
+        return $data instanceof PriceCollectionProductValue &&
+            AttributeTypes::BACKEND_TYPE_PRICE === $data->getAttribute()->getBackendType() &&
+            'indexing' === $format;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getNormalizedData(ProductValueInterface $productValue)
+    {
+        $currencyIndexedPrices = [];
+
+        foreach ($productValue->getData() as $price) {
+            $currencyIndexedPrices[$price->getCurrency()] = (string) $price->getData();
```

Comment:
@SamirBoulil if $price->getCurrency() is empty, it's not necessary to normalize it